src/core/tests/type_prop/binary_elementwise.cpp

```diff
@@ -468,164 +468,175 @@ shared_ptr<op::v1::Reshape> createReshapeSubgraph(PartialShape param_shape,
 }
 }  // namespace
 
-TEST(type_prop, interval_value_propagation_add) {
+TEST(type_prop, interval_value_propagation_add_rhs) {
     PartialShape op_shape{Dimension(-1), Dimension(2, -1), Dimension(-1, 6), Dimension(7, 10), Dimension(7, 10), 5};
     const auto const_op = op::Constant::create(element::f32, {6}, {2, 3, 4, 5, -5, 6});
-    {  // const rhs
+    // const rhs
+    const auto reshape = createReshapeSubgraph<op::v1::Add>(op_shape, const_op);
+    EXPECT_EQ(reshape->get_element_type(), element::f32);
+    EXPECT_EQ(reshape->get_output_partial_shape(0),
+              PartialShape({-1, /* Dimension(5, -1) */ -1, Dimension(4, 10), Dimension(12, 15), Dimension(2, 5), 11}));
+}
+
+TEST(type_prop, interval_value_propagation_add_lhs) {
+    PartialShape op_shape{Dimension(-1), Dimension(2, -1), Dimension(-1, 6), Dimension(7, 10), Dimension(7, 10), 5};
+    const auto const_op = op::Constant::create(element::f32, {6}, {2, 3, 4, 5, -5, 6});
+    // const lhs
+    const auto reshape = createReshapeSubgraph<op::v1::Add>(op_shape, const_op, false);
+    EXPECT_EQ(reshape->get_element_type(), element::f32);
+    EXPECT_EQ(reshape->get_output_partial_shape(0),
+              PartialShape({-1, /* Dimension(5, -1) */ -1, Dimension(4, 10), Dimension(12, 15), Dimension(2, 5), 11}));
```

Comment:
/* Dimension(5, -1) */ can be removed?
 or add more detailed comment why expecting this dimension, check all similar cases.

Reply:
Removed. Ticket for the discussed bounds evaluation improvements: 95040